projects/plugins/jetpack/extensions/blocks/videopress/edit.js

```diff
@@ -1049,6 +1050,35 @@ export const VpBlock = props => {
 		} ),
 	} );
 
+	const getSandboxScripts = useCallback( () => {
```

Comment:
I think in this case, it could be replaced with a `useMemo` for `sandboxScripts`. With that, the function isn't called on every render, as the initial scripts array will likely be the same.

Reply:
I agree that a `useMemo` would make more sense here.

But it would run before the conditional return of the `shouldRenderLoadingBlock`, and it wouldn't be needed in that scenario, even if the code is fixed to not break depending on the value of the `scripts`.

In terms of performance, I think we would have pros and cons in both approaches, but in the end it wouldn't be significant.

WDYT? Does it make sense for you?

Reply:
Indeed, we can keep it that way then. I'll approve it, thanks for the follow-up!